lib/preload.js

```diff
@@ -24,7 +24,6 @@ const DOCKER_IMAGE_TAG = 'balena/balena-preload';
 const DISK_IMAGE_PATH_IN_DOCKER = '/img/balena.img';
 const SPLASH_IMAGE_PATH_IN_DOCKER = '/img/balena-logo.png';
 const DOCKER_STEP_RE = /Step (\d+)\/(\d+)/;
-const EXIT_CAUSED_BY_SIGINT = 137;
```

Comment:
I gather that code 137 is for SIGKILL, not SIGINT, and it could be caused by the container / system running out of memory. I assume we would want to report OOM SIGKILL through to the CLI process, hence removing the test `data.StatusCode !== EXIT_CAUSED_BY_SIGINT`.